### PR TITLE
Minor imprs for osgi config factory

### PR DESCRIPTION
**Pull request by krystian-panek-wttech**

### CODE REVIEW

File: pkg/osgi_config_manager.go [hunk 1]
```diff
@@ -18,23 +18,24 @@ type OSGiConfigManager struct {
...
-	}
-	if len(tokens) == 1 {
-		return tokens[0], ""
+func (cm *OSGiConfigManager) SplitFactoryPID(pid string) (string, string) {
+	parts := strings.Split(pid, osgi.ConfigAliasSeparator)
+	if len(parts) != 2 {
+		log.Fatalf("cannot parse OSGi config factory PID '%s'", pid)
 	}
-	return "", ""
+	return parts[0], parts[1]
```
Comment (piotr-andruszkiewicz-wttech) [inline pkg/osgi_config_manager.go:38]: So in case when `len(parts) == 1` we log the message and then proceed to panic?

File: pkg/osgi/config.go [hunk 2]
```diff
@@ -50,8 +50,8 @@ func (c ConfigListItem) PropertyValues() map[string]any {
 // CID Extracts the constant ID, which is used to find config. See FPIDDummy for explanation.
 func (c ConfigListItem) CID() string {
```
Comment (piotr-andruszkiewicz-wttech) [inline pkg/osgi/config.go:51]: This could also be renamed to Alias

File: pkg/osgi_config_manager.go (hunk 1 above)
Comment (krystian-panek-wttech) [reply pkg/osgi_config_manager.go:38]: no, before we have now `if` which does not invoke this method then at all;

the code is more strict

File: pkg/osgi/config.go (hunk 2 above)
Comment (krystian-panek-wttech) [reply pkg/osgi/config.go:51]: right, missed

**Pull request merged**